.travis.yml

```diff
@@ -29,15 +35,17 @@ services:
 
 before_install:
     - echo '' > ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini
-    - echo "extension = mongo.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
-    - echo "zend.enable_gc = 0" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
     - echo "memory_limit=4096M" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
+    - if [ $TRAVIS_PHP_VERSION != "7.0" ]; then echo "extension = mongo.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini; fi
 
     - composer self-update
     - composer install --prefer-source --no-interaction
-    - composer require doctrine/mongodb-odm="1.0.*@dev" --no-update
-    - composer require jmikola/geojson="~1.0" --no-update
-    - composer update doctrine/mongodb-odm jmikola/geojson --prefer-source --no-interaction $COMPOSER_FLAGS
+
+#    - if [ $TRAVIS_PHP_VERSION != "7.0" ]; then composer require doctrine/mongodb-odm="1.0.*@dev" --no-update; fi
+#    - if [ $TRAVIS_PHP_VERSION != "7.0" ]; then composer require jmikola/geojson="~1.0" --no-update; fi
+#    - if [ $TRAVIS_PHP_VERSION != "7.0" ]; then composer update doctrine/mongodb-odm jmikola/geojson --prefer-source --no-interaction; fi
+
+    - if [ $TEST_SUITE = "javascript" ]; then travis/prepare-to-javascript-tests; fi
```

Comment:
Don't waste resources to build apache, as we changed to PHP 5.5, use built-in webserver!

```yml
before_script:
    - php -S localhost:8000 -t web &> /dev/null;

    - curl http://selenium-release.storage.googleapis.com/2.47/selenium-server-standalone-2.47.1.jar > selenium.jar
    - java -jar selenium.jar > /dev/null &
    - sleep 5
```
Later add:
```yml
# behat.yml.dist
default:
     extensions:
         Behat\MinkExtension:
            base_url: 'http://localhost:8000/app_test.php'